bindings_generator/src/api.rs

```diff
@@ -3,12 +3,14 @@ use serde_json;
 
 pub struct Api {
     pub classes: Vec<GodotClass>,
+    pub api_underscore: Vec<String>,
 }
 
 impl Api {
     pub fn new() -> Self {
         let mut api = Api {
             classes: serde_json::from_slice(get_api_json()).expect("Failed to parse the API description"),
+            api_underscore : Default::default(),
```

Comment:
The space between `api_underscore` and `:` will make `cargo fmt` unhappy :smile: